### PR TITLE
Added "What's New in DevTools (Microsoft Edge 99)"

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools?branch=pr-en-us-1779

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:35]: Instead of just linking to this one doc, maybe link to all 3 that are related:
```suggestion
See:
- [Map the processed code to your original source code, for debugging](../../../javascript/source-maps.md),
- [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md).
- [Securely debug original code by using Azure Artifacts symbol server source maps](/javascript/consume-source-maps-from-azure.md).
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:67]: I don't really get why this is italicized. If it's in our repo, then I'd recommend just doing:
```suggestion
* [Edge DevTools extension for Visual Studio](../../../../visual-studio/index.md#edge-devtools-extension-for-visual-studio).
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:76]:
```suggestion
Since [Microsoft Edge 88](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2020/11/devtools#composited-layers-are-now-in-3d-view), the **3D View** tool has provided an improved version of the Layers tool. Now, in Microsoft Edge 99, the stand-alone Layers tool is being removed. The **Layers** tool will display a link that redirects you to the **Composited Layers** view within the **3D View** tool.  You can find all the functionality from the **Layers** tool there.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:78]: Was using Focus Mode for this screenshot an intentional choice? I feel like it's unnecessarily verbose to talk about Focus Mode and Quick View when they're not really relevant to this announcement. I'd rather we just use the vanilla DevTools UI for this screenshot and draw the red highlight boxes around the Layers tool in the drawer

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:101]: What do you think about doing these 2 images side-by-side? Are we not doing side-by-side images any more? I feel like in this case, it might make it clearer what the changes are to the call stack but I'll defer to you

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:130]: I would prefer if we can land [this PR](https://github.com/MicrosoftDocs/edge-developer/pull/1784) and then link to it from here but if we can't make it work in time, we can always add the link later and just have the link to the blog for now

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:154]: Maybe for this example about what assistive tech will announce, we should say "selected" since in the screenshot, that radio button is selected.
```suggestion
In previous versions of Microsoft Edge, assistive technology would only announce the radio button text in the 3D View tool, without announcing the group name that the button belongs to.  In Microsoft Edge version 99, this has been fixed.  For example, when focus is on the **Use screen texture** radio button, assistive technology will now announce: "Color type radio group, Use screen texture, radio button, selected".
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:103]: Please crop the window title bar out of this screenshot. Here's an example of what the image should look like when cropped: 
image

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:115]: Please crop the title bar out of this screenshot as well

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:171]: I don't think we need this detail about normalizing absolute source URLs. It feels to me like an implementation detail.
```suggestion
In the **Page** tab of the **Sources** tool, the tree of source folders is now improved, with less clutter in the naming and outline of folders.  Unneeded path prefixes such as `../` and `./` have been removed.  The tree structure has been simplified by merging equivalent duplicate folders. 
```

**Review by zoherghadyali — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:101]: https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#columns - "While the columns extension still works, we no longer recommend it for creating custom layouts. We've found that many custom column layouts have accessibility issues or otherwise violate Docs style guidelines. Don't create custom layouts. Use standard Docs features."

Side-by-side image presentation was implemented by :::image::: custom markup tagging.  To make the articles legible/readable by simply scrolling, instead of mode-switching to having to pop-out / zoom the images, and as part of the Initiative: _Improve the content authoring / Content Developer experience_, we switched from ::: custom markup tagging to standard Markdown tagging, and no longer use ::: column tagging or ::: image tagging.

Now we present content by figuring out ways to leverage simple standard Markdown as much as possible, rather than relying on custom markup and column layout and image taging.  eg one benefit of this is color syntax coloring of alt-text while authoring in .md files in VS Code, producing better alt-text.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:103]: What is the motivation or objective for removing the title bar from these (the Light mode) captures?  If the objective is to not show the URL, I'd rather erase the URL from the title bar rather than erase the entire title bar.  If the objective is to never show title bars, we need to meet to set new conventions for screenshot standards for Win11 screenshots.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:154]: Your change makes more sense than what I copied from the work item.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:78]: The work item instructions specified Focus Mode as a means of showing old + new tool together but I think regular mode supports the same layout.  I mentioned my trying this, to mgmt today, and that I added the clarifying statement.  Will msg you + mgmt b/c this relates to timing decisions about screenshots standards/style guidelines.

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:78]: Re-created w/o focus mode

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:67]: I will remove it but it is there for solid reason: because w/o it, the Content Developer (writer) can't tell where the link goes, while looking at .md tagging view.  Prevents being lost in navigation.  Warns author/reader where they'll end up after traversing the link: not an article's top, but an h2 halfway down in a page, which is not visibly clear after arrival, until scroll to top of the destination page - quite disorienting.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:130]: I did a rewrite, Writer/Editor pass, on that PR, have to fix a bold tag, wish to have Patrick's input on differentiation of benefits of source maps vs deminification vs ADO secure source maps server, in top of that article.  Near-ready to merge that PR.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:114]: Erased URL.  Is that sufficient, or must the titlebar be removed entirely?  
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools?branch=pr-en-us-1779#use-your-source-maps-to-unminify-performance-profiles

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:102]: @zoherghadyali  - Erased URL from the titlebar.  Sufficient?  Or is a requirement that the titlebar be entirely removed?
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools?branch=pr-en-us-1779#use-your-source-maps-to-unminify-performance-profiles

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:91]:
```suggestion
In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack did not use source maps to unminify the call stack, even if you hosted the source maps on your server.
```

Comment (Reezaali) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:91]: @zoherghadyali - please review my suggestion.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:93]: Title says "Use your source maps to unminify performance profiles" but here we talk about unminifying the call stack. To be consistent, I think we should change the title. 

@zoherghadyali?

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:112]:
```suggestion
After clicking **Unminify**, the unminified call stack in the **Performance** tool now shows those functions labeled with meaningful names: **invokeFunc (b)**, **executeAction (O)**, **endBatch (Xt)**, and **runReactions (bn)**:
```

Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:112]: @zoherghadyali @captainbrosset @Reezaali - change "call stack" to "flame chart" in this What's New item?  Like per captainbrosset's comments on the Unminify PR.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:91]: Suggestion v3:

Recording a performance profile in the **Performance** tool produces a minified call stack.  In previous versions of Microsoft Edge, the call stack (flame chart) didn't use source maps to display your original function names, even if you hosted the source maps on your server.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:109]: Works for me. Thanks Michael!

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:105]: Works for me! Thanks Michael!

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md]: To match with the docs for this feature, let's just do: 

Recording a performance profile in the Performance tool produces a minified flame chart. In previous versions of Microsoft Edge, the flame chart didn't use source maps to display your original function names, even if you hosted the source maps on your server.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md]: Yes, I agree with matching the docs for this feature but let's try to keep the title concise. I propose:

title: Use your source maps to display original function names in performance profiles

Starting in Microsoft Edge version 99, you can display your original function names in a performance profile, as follows:

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (zoherghadyali) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md]: Yes, I agree with that. Flame chart is the correct term

**Review by zoherghadyali — APPROVED** [review]

File: microsoft-edge/visual-studio/index.md
Comment (codepo8) [inline microsoft-edge/visual-studio/index.md:160]: Probably replace "open" with "available" as by default, the tab is available but not selected.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (codepo8) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:12]: As there is no announcement about the VS Code extension, but the VS one in this what's new, this feels weird. How about we make it future proof by changing it to:

To try the latest features of DevTools and extension for editors using them, read these announcements.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (codepo8) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:15]: This would also be a good opportunity to promote the new repo: 

"If you want to report a problem with DevTools or ask for a new feature, you can [file an issue](https://github.com/MicrosoftEdge/DevTools/issues/new/choose) on the [GitHub Edge DevTools repository](https://github.com/MicrosoftEdge/DevTools)."

**Review by codepo8 — CHANGES_REQUESTED** [review]
Pending the comments I added this LGTM

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (robpaveza) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:28]: I think it would be nice if we linked this line, or some part of it, to the same location as the 2nd bullet in the See Also section below. E.g.:

```md
1. [Publish your source maps](../../../javascript/publish-source-maps-to-azure.md) to Azure Artifacts Symbol Server.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md
Comment (erdraud) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md:79]: Realizing that "Layers tool has been removed and contains a link..." doesn't really make sense. Let's change it to "Now, in Microsoft Edge 99, the functionality in the standalone **Layers** tool has been removed and instead contains a link redirecting you to the **Composited Layers** view within the **3D View tool. You can find all the features previously in the **Layers** tool and more in the **3D View** tool.

**Review by zoherghadyali — APPROVED** [review]

**Review by codepo8 — APPROVED** [review]
LGTM

**Pull request merged**
